src/Processing/ImportProcessingService.php

```diff
@@ -286,6 +260,51 @@ protected function processElement(string $configName, array $importDataRow, Reso
         }
     }
 
+    /**
+     * Process transformations for an element
+     * 
+     * @param MappingConfiguration[] $mapping Array of mapping configurations.
+     * @param-out mixed &$currentMapping Output variable that will contain the current mapping.
+     * @throws \Throwable If the processing or a transformation fails
+     * @return void
```

Comment:
Thank you for applying the suggestions above!

```suggestion
     *
     * @throws \Throwable If the processing or a transformation fails
```
just one final code style changes and looks ready to be merged:
- The `@throw` part usually has an extra empty line and is stick to the bottom.
eg.
https://github.com/pimcore/pimcore/blob/df14e025d3db615fc75b190e3c9f3cc1d68d7304/lib/Twig/Extension/Templating/HeadMeta.php#L264-L268
https://github.com/pimcore/pimcore/blob/df14e025d3db615fc75b190e3c9f3cc1d68d7304/lib/Console/Traits/Timeout.php#L74-L77
- The `@return` is mostly used/kept for when it lacked the php native ones, so if there are no particular comments/notes for it, it can actually be removed.